### DISCUSSION TITLE
浴霸以及晾衣架设备的支持不完善

### DISCUSSION THREAD

**Discussion by erchuyi** [XiaoMi/ha_xiaomi_home#111] [General]
浴霸：风暖，吹风，换气等功能单独开关正常，ha 状态也能更新。

设备：易来舒适暖浴霸 pro

问题：但是如果在浴霸风暖，吹风，换气等功能开启的状态下，按浴霸遥控器上的“休眠”功能，浴霸会关闭所有功能，米家 app 内也会显示关闭，但是 ha 状态中风暖，吹风，换气等不会更新状态为关闭。

晾衣机： 晾衣机的上升下降功能在添加到 homekit 无法很好的控制。

设备：米家智能晾衣机 pro 白月光

**Comment by SusanPhevos** [comment]
浴霸的问题可能是设备没有上报状态改变。请提一个issue，附加问题复现时的Home Assistant日志，我们看一下。

**Reply by erchuyi** [reply]
您好，我是为了 xiaomihome 插件注册的 GitHub，还不太熟练，具体情况就是浴霸面板上有一个“待机”按钮，该按钮会让浴霸除灯光外的所有功能关闭，点击“待机”后，原本开启的风暖关闭了，在米家 App 里也显示关闭，但是在 ha 里还是开启状态。

但是如果我单独的去控制风暖的开启关闭，ha 里的状态会实时改变。

我看了 ha 日志内，我点了“待机”按钮后完全没有生成任何日志。
